tests/blackbox/stratis_cli_cert.py

```diff
@@ -20,12 +20,66 @@
 import time
 import unittest
 
-from testlib.utils import exec_command, exec_test_command, process_exists
+from testlib.utils import exec_command, exec_test_command, process_exists, p_n, fs_n
 from testlib.stratis import STRATIS_CLI, StratisCli, clean_up
 
 DISKS = []
 
 
+def make_test_pool():
+    """
+    Create a test pool that will later get destroyed
+    :return: Name of the created pool
+    """
+    pool_name = p_n()
+    exec_test_command([STRATIS_CLI, "pool", "create", pool_name, DISKS[0]])
+    return pool_name
+
+
+def make_test_filesystem(pool_name):
+    """
+    Create a test filesystem that will later get destroyed
+    :param pool_name: Name of a test pool
+    :return: Name of the created filesystem
+    """
+    filesystem_name = fs_n()
+    assert (
+        exec_test_command(
+            [STRATIS_CLI, "filesystem", "create", pool_name, filesystem_name]
+        )[0]
+        == 0
+    )
+    return filesystem_name
+
+
+def test_command(  # pylint: disable=bad-continuation
+    self, command_line_args, exp_exit_code, exp_stderr_is_empty, exp_stdout_is_empty
+):
+    """
+    Execute a test command and make assertions about the exit code, stderr, and stdout
+    :param self: A test object
+    :param command_line_args: A list of command line arguments needed to execute the Stratis command
+    :type command_line_args: List of str
+    :param exp_exit_code: The expected exit code, 0, 1, or 2
+    :param exp_stdout_is_empty: A boolean representing whether or not we expect stdout to be empty
```

Comment:
Make it easier for the users to figure out what these two parameters mean:

```
:param bool exp_stdout_is empty: True if sdout is expected to be empty, otherwise False
```

Same for the other one.

Note that the doc comments are in reverse order with the order of the parameters, too. That could really confuse a reader.